pkg/nsx/services/subnetport/builder.go

```diff
@@ -24,9 +24,29 @@ import (
 )
 
 var (
-	String = common.String
+	String                      = common.String
+	defaultContainerMacPoolName = "DefaultContainersMacPool"
```

Comment:
need to check if only this mac pool is used to allocate mac address for subnetport.

Reply:
Checked with NSX side, defaultContainerMacPoolName is the one used for SubnetPort address allocation. This is an mp api, but there is no policy api doing the same thing.

Reply:
Do we confirm only need to check this mac Pool?

Reply:
Yes, Nilesh confirm that.